priv/db-actions/up/create-view.sql.eex

```diff
@@ -0,0 +1,53 @@
+CREATE MATERIALIZED VIEW "<%= view_name %>" AS
```

Comment:
Should we CREATE OR REPLACE? Just to be safe?

Reply:
Let it blow up.